forge-gui/res/cardsfolder/upcoming/angelic_sleuth.txt

```diff
@@ -0,0 +1,10 @@
+Name:Angelic Sleuth
+ManaCost:2 W
+Types:Creature Angel Advisor
+PT:2/3
+K:Flying
+

```

Comment:
Empty lines would need to be removed from the final script.

Reply:
Done, will remember to do so in future too!